plan/dag_plan_test.go

```diff
@@ -1040,16 +1048,17 @@ func (s *testPlanSuite) TestRefine(c *C) {
 		},
 		{
 			sql:  `select a from t where c = 'hanfei'`,
-			best: "TableReader(Table(t))->Sel([eq(cast(test.t.c), cast(hanfei))])->Projection",
+			best: "IndexReader(Index(t.c_d_e)[[0,0]])->Projection",
```

Comment:
Why is this test changed ?

Reply:
use test set sc.IgnoreTruncate to true,
SelectStmt reset this when executing, we only compile statement and reuse the sc.IgnoreTruncate set be UseStmt here, so this case changed.